cli/cmd/get.go

```diff
@@ -78,6 +78,9 @@ func runGet(cmd *cobra.Command, args []string) (string, error) {
 
 	resourcesRes, err := getResourcesResponse(appName)
 	if err != nil {
+		if strings.HasSuffix(err.Error(), "is not deployed") {
```

Comment:
Can you add a comment at every location where this string must match (i.e. where it is defined and where it is pattern matched)? Similar to https://github.com/cortexlabs/cortex/blob/master/cli/cmd/cluster.go#L145